python/lsst/dax/imgserv/cli.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+# Copyright 2017 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+
+'''
+This module implements the command line interface (CLI) to ImageServ.
+
+@author: Brian Van Klaveren, SLAC
+@author: Kenny Lo, SLAC
+
+'''
+
+import os
+import json
+
+import click
+
+import lsst.log as log
+import lsst.daf.persistence as dafPersist
+
+from .locateImage import (
+        image_open, W13DeepCoaddDb, W13RawDb, W13CalexpDb
+)
+
+from .image import Image
+from .dispatch import Dispatcher
+from .hashutil import Hasher
+
+ROOT = os.path.abspath(os.path.dirname(__file__))
+
+@click.command()
+@click.option("--config", help="Specify config file.")
+@click.option("--in_req", help="Specify query input.")
```

Comment:
I think it's technically an argument, not an option.

An `--config` or a theoretical `--verbose` mode would be a `click.option`.

`@click.argument('query_input', type=click.Path(exists=True))`
`@click.argument('output', type=click.Path()) # Maybe exists=True if you want Click to validate`

In the future, it may be an option, as the CLI should be able to accept input _other than JSON files_.